### PR TITLE
add call control endpoint

### PR DESCRIPTION
**Pull request by lunarfyre7**

### CODE REVIEW

File: lib/telnyx/api_operations/nested_resource.rb [hunk 1]
```diff
@@ -12,11 +12,16 @@ module APIOperations
     module NestedResource
       def nested_resource_class_methods(resource, path: nil, operations: nil)
         path ||= "#{resource}s"
+        if path.is_a? Array
+          path = path.map { |el| CGI.escape(el) }.join('/')
+        else
+          path = CGI.escape path
+        end
```
Comment (davidkovsky) [inline lib/telnyx/api_operations/nested_resource.rb]: What do you think of removing the if/else like this?

`Array(path).map { |el| CGI.escape(el) }.join('/')`

File: lib/telnyx/calls.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+module Telnyx
+  class Calls < APIResource
+    extend Telnyx::APIOperations::Create
+    extend Telnyx::APIOperations::NestedResource
+
+    ACTIONS = %w[reject answer hangup bridge speak fork_start fork_stop 
+    gather_using_audio gather_using_speak playback_start 
+    playback_stop record_start record_stop send_dtmf transfer]
+
```
Comment (davidkovsky) [inline lib/telnyx/calls.rb]: I'm surprised rubocop didn't want to modify the indentation above?

File: lib/telnyx/api_operations/nested_resource.rb (hunk 1 above)
Comment (AkshathaC17) [reply lib/telnyx/api_operations/nested_resource.rb]: +1

File: lib/telnyx/calls.rb (hunk 2 above)
Comment (lunarfyre7) [reply lib/telnyx/calls.rb]: The old rubocop config seems to have removed, I didn't want to rerun it with the default config and change everything.

File: lib/telnyx/api_operations/nested_resource.rb (hunk 1 above)
Comment (lunarfyre7) [reply lib/telnyx/api_operations/nested_resource.rb]: Much cleaner!

**Pull request merged**